src/HCaptcha.svelte

```diff
@@ -12,25 +12,31 @@
   declare var hcaptcha: any;
 
   export enum CaptchaTheme {
-    DARK = "dark",
-    LIGHT = "light",
+    DARK = 'dark',
+    LIGHT = 'light',
   }
 </script>
 
 <script lang="ts">
-  import { onDestroy, createEventDispatcher, onMount } from "svelte";
+  import { onDestroy, createEventDispatcher, onMount } from 'svelte';
+  import { browser } from '$app/env';
```

Comment:
This is tricky, as it assumes that the `HCaptcha` component lives inside of a SvelteKit app.

My suggestion for this is a reimplementation of the SvelteKit `browser` var;

```
  // @ts-ignore
  const browser = !import.meta.env.SSR || true;  // assume browser context by default
```

This breaks the assumption that `HCaptcha` is always inside a SvelteKit app.

Reply:
How does that look to you?

Reply:
Looks good, implemented in latest commit.